tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -87,6 +87,92 @@ public function userHasBeenEditedUsingTheGraphApi(
 		$this->featureContext->theHttpStatusCodeShouldBe(200);
 	}
 
+	/**
+	 * @When /^the user "([^"]*)" changes the email of user "([^"]*)" to "([^"]*)" using the Graph API$/
+	 * @When /^the user "([^"]*)" tries to change the email of user "([^"]*)" to "([^"]*)" using the Graph API$/
+	 *
+	 * @param string $byUser
+	 * @param string $user
+	 * @param string $email
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 * @throws Exception
+	 */
+	public function theUserChangesTheEmailOfUserToUsingTheGraphApi(string $byUser, string $user, string $email): void {
+		$this->editUserUsingTheGraphApi($byUser, $user, null, null, $email);
+	}
+
+	/**
+	 * @When /^the user "([^"]*)" changes the display name of user "([^"]*)" to "([^"]*)" using the Graph API$/
+	 * @When /^the user "([^"]*)" tries to change the display name of user "([^"]*)" to "([^"]*)" using the Graph API$/
+	 *
+	 * @param string $byUser
+	 * @param string $user
+	 * @param string $displayName
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 * @throws Exception
+	 */
+	public function theUserChangesTheDisplayNameOfUserToUsingTheGraphApi(string $byUser, string $user, string $displayName): void {
+		$this->editUserUsingTheGraphApi($byUser, $user, null, null, null, $displayName);
+	}
+
+	/**
+	 * @Then /^the user "([^"]*)" should have information with these key and value pairs:$/
+	 *
+	 * @param string $user
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws Exception
+	 * @throws GuzzleException
+	 * @throws JsonException
+	 */
+	public function theUserShouldHaveInformationWithTheseKeyAndValuePairs(string $user, TableNode $table): void {
+		$rows = $table->getHash();
+		foreach ($rows as $row) {
+			$key = $row['key'];
+			$expectedValue = $row['value'];
+		}
+		$this->adminHasRetrievedUserUsingTheGraphApi($user);
+		$responseValue = $this->featureContext->getJsonDecodedResponse($this->featureContext->getResponse())[$key];
+		Assert::assertEquals(
+			$expectedValue,
+			$responseValue
+		);
+	}
+
+	/**
+	 * Edits the user information
+	 *
+	 * @param string $byUser
+	 * @param string $user
+	 * @param string|null $userName
+	 * @param string|null $password
+	 * @param string|null $email
+	 * @param string|null $displayName
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 */
+	public function editUserUsingTheGraphApi(string $byUser, string $user, string $userName = null, string $password = null, string $email = null, string $displayName = null): void {
+		$userId = $this->featureContext->getAttributeOfCreatedUser($user, 'id');
+		$response = GraphHelper::editUser(
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getStepLineRef(),
+			$byUser,
+			$this->featureContext->getPasswordForUser($byUser),
+			$userId,
+			$userName,
+			$password,
+			$email,
+			$displayName
+		);
+		$this->featureContext->setResponse($response);
+	}
```

Comment:
Return response from helper functions and set the response in the `When` step function
```suggestion
	public function editUserUsingTheGraphApi(string $byUser, string $user, string $userName = null, string $password = null, string $email = null, string $displayName = null): void {
		$userId = $this->featureContext->getAttributeOfCreatedUser($user, 'id');
		return GraphHelper::editUser(
			$this->featureContext->getBaseUrl(),
			$this->featureContext->getStepLineRef(),
			$byUser,
			$this->featureContext->getPasswordForUser($byUser),
			$userId,
			$userName,
			$password,
			$email,
			$displayName
		);
	}
```